tests/Kros.AspNetCore.Tests/Authorization/GatewayAuthorizationMiddlewareShould.cs

```diff
@@ -249,40 +326,44 @@ private static (IHttpClientFactory, GatewayAuthorizationMiddleware) CreateMiddle
         private static (IHttpClientFactory, GatewayAuthorizationMiddleware) CreateMiddleware(
             HttpStatusCode statusCode,
             TimeSpan offset,
-            List<string> ignoredPathForCache)
+            List<string> ignoredPathForCache,
+            GatewayJwtAuthorizationOptions options = null)
         {
-            var gatewayJwtAuthorizationOptions = new GatewayJwtAuthorizationOptions()
+            if (options is null)
             {
-                AuthorizationUrl = "http://authorizationservice.com",
-                HashAuthorizationUrl = "http://hashauthorizationservice.com",
-                HashParameterName = "hash",
-                CacheSlidingExpirationOffset = offset
-            };
-            gatewayJwtAuthorizationOptions.IgnoredPathForCache.AddRange(ignoredPathForCache);
+                options = new GatewayJwtAuthorizationOptions()
+                {
+                    AuthorizationUrl = AuthorizationUrl,
+                    HashAuthorizationUrl = HashAuthorizationUrl,
+                    HashParameterName = "hash",
+                    CacheSlidingExpirationOffset = offset
+                };
+            }
+            options.IgnoredPathForCache.AddRange(ignoredPathForCache);
 
             var httpClientFactory = Substitute.For<IHttpClientFactory>();
 
-            FakeHttpMessageHandler fakeHttpMessageHandler = CreateFakeHttpMessageHandler(statusCode, gatewayJwtAuthorizationOptions);
+            FakeHttpMessageHandler fakeHttpMessageHandler = CreateFakeHttpMessageHandler(statusCode);
             var fakeHttpClient = new HttpClient(fakeHttpMessageHandler);
             httpClientFactory.CreateClient("JwtAuthorizationClientName").Returns(fakeHttpClient);
 
-            var middleware = new GatewayAuthorizationMiddleware((c) => Task.CompletedTask, gatewayJwtAuthorizationOptions);
+            var middleware = new GatewayAuthorizationMiddleware((c) => Task.CompletedTask, options);
 
             return (httpClientFactory, middleware);
         }
 
-        private static FakeHttpMessageHandler CreateFakeHttpMessageHandler(HttpStatusCode statusCode, GatewayJwtAuthorizationOptions gatewayJwtAuthorizationOptions)
+        private static FakeHttpMessageHandler CreateFakeHttpMessageHandler(HttpStatusCode statusCode)
         {
             var responses = new List<KeyValuePair<FakeHttpMessageHandler.HttpRequestFilter, HttpResponseMessage>>();
             responses.Add(new KeyValuePair<FakeHttpMessageHandler.HttpRequestFilter, HttpResponseMessage>(
-                (req) => req.RequestUri.AbsoluteUri.Contains(gatewayJwtAuthorizationOptions.AuthorizationUrl),
+                (req) => req.RequestUri.AbsoluteUri.Contains(AuthorizationUrl),
                 new HttpResponseMessage()
                 {
                     StatusCode = statusCode,
                     Content = new StringContent("MyJwtToken")
```

Comment:
Nebolo by vhodne dat toto ako konstantu? Lebo ak by tu nastala zmena, tak je to na vela miestach v testoch...